src/selectedtests/task_mappings/mappings.py

```diff
@@ -35,7 +35,6 @@ def create_task_mappings(
         evergreen_project: str,
         start_date: datetime,
         end_date: datetime,
-        org_name: str,
```

Comment:
I removed org_name here because I realized we can get it from the /projects endpoint. See this ticket for more info: https://jira.mongodb.org/browse/EVG-6878.

Reply:
That'll definitely work in that we do need less information from the user now which is good. However, I'm not super in love with having to iterate over all the projects in evergreen to find the info for the project we're analyzing every time we run the script. 

I think this is a good workaround for getting the needed info but that we should keep the request open for evergreen to expose this information on the ```projects/{projectId}``` endpoint. What do others think?

Reply:
If iterating over all the projects had a high performance cost, I would be willing to keep the request open. But since there's a workaround, the `projects/{projectId}` endpoint change doesn't seem super high priority to me.

Reply:
I think there are around around 300 projects, so iterating over it should be ok. I didn't realize the authenticated user had to have permissions for each project to hit that projects endpoint. That is annoying.

Reply:
Created this ticket to follow up on this comment outside of this PR: https://jira.mongodb.org/browse/TIG-2091